### ISSUE TITLE
Ignore Button on Nodes

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#464]
## Overview
We need an ignore button to mark nodes ignored. Users requested this because some of our scans report false positives. The button must log when an issue is ignored or unignored, similar to how a node is logged equalified or unequalified.

The ignore action should be done on a single property page, since it is a universal action.

We discussed either included a list of nodes on the property page, or a search that allowed you to search for nodes and add them of a list of ignored nodes. You should be also able to remove them from the ignored list. It may be helpful to include the time when a node was ignored.

That UX may be changed in Version 2.

## Bountied
Since this is a bountied issue, Assignees should post an estimate that is then approved as per this note in the handbook: https://github.com/EqualifyEverything/handbook?tab=readme-ov-file#how-are-bounties-budgeted

**Comment by bbertucc** [comment]
Adding @heythisischris to budget work on this. He's open for anyone to pair with him.

**Comment by bbertucc** [comment]
We started to discuss this in #323. We decided not to include the explanation because it's seemingly penalizing users for using a feature. We don't want to create anti-user features like that. Plus, the user will have a record of when things were ignored or not.

**Comment by bbertucc** [comment]
@heythisischris can you add an estimate to this?

**Comment by heythisischris** [comment]
@bbertucc Yes I can- I estimate this will take between 4-8 hours. Let's start with 4 hours.

**Comment by bbertucc** [comment]
Great! Approved @heythisischris

**Comment by bbertucc** [comment]
This has been added to https://github.com/EqualifyEverything/equalify-v2-dashboard
